utils/update_translation_stats.py

```diff
@@ -58,18 +57,17 @@ def generate_translation_stats(po_dir, output_file):
         sys.stdout.flush()
 
         try:
-            # We need shell=True, otherwise we get "No such file or directory".
-            stats_output = check_output(
-                ['pocount ' + subdir + ' --csv'],
+            stats_output = subprocess.check_output(
+                ['pocount', '--csv', subdir],
                 encoding='utf-8',
                 stderr=subprocess.STDOUT,
-                shell=True)
+                )
             if 'ERROR' in stats_output:
-                print('\nError running pocount:\n' + stats_output.split('\n', 0)
-                      [0]) + '\nAborted creating translation statistics.'
+                print('\nError running pocount:\n' + stats_output.split('\n', 1)[0] +
```

Comment:
Hmm, this is strange. I don't know what the original intent was, but printing only the first line of the output doesn't seem to make sense. At least print all lines containing "ERROR"?

Reply:
My final suggestion is to use separate stderr (6f22d8178e566def9e34cc02bf3e013e489f5307 in [branch update_translations_stats_speedup](https://github.com/SimonHeimberg/widelands/commits/update_translations_stats_speedup)) Shall I include the commits of speedup pr in this one? Or make the 2nd pr later?

Reply:
I think it would be OK if you just printed the whole `stats_output` in case of error here, and did the stderr thing in the later PR.